managed/models/database.go

```diff
@@ -728,6 +728,22 @@ var databaseSchema = [][]string{
 		)`,
 	},
 	66: {
+		`UPDATE agents
+		SET log_level = 'error'
+		WHERE log_level = 'fatal'
+		AND agent_type IN (
+			'node_exporter',
+
+			'mysqld_exporter',
+			'qan-mysql-perfschema-agent',
+			'qan-mysql-slowlog-agent',
```

Comment:
no need to decrease the log level for QAN agents

Reply:
done